test/analytics/analytics-on-redshift/lambda/scan-metadata-workflow/store-metadata-into-ddb.test.ts

```diff
@@ -44,151 +44,474 @@ describe('Lambda - store the metadata into DDB from Redshift', () => {
+                  category: 'category',
+                  platform: ['APP', 'ANDROID', 'IOS', 'WEB'],
+                  valueEnum: [
+                    {
+                      count: 40,
+                      value: 'value1',
+                    },
+                    {
+                      count: 50,
+                      value: 'value2',
+                    },
+                    {
+                      count: 50,
+                      value: 'value',
+                    },
+                    {
+                      count: 20,
+                      value: 'value3',
+                    },
+                  ],
+                },
               },
             },
           },
           {
             PutRequest: {
               Item: {
-                appId: 'appId',
-                category: 'category',
-                createAt: 1695120657125,
-                deleted: false,
-                description: '',
-                displayName: '',
-                hasData: true,
-                id: 'projectId#appId#eventName#userattributename',
-                metadataSource: 'custom',
-                name: 'userattributename',
-                operator: '',
-                parameterType: 'Private',
-                platform: ['ANDROID', 'IOS'],
-                prefix: 'USER_ATTRIBUTE#projectId#appId',
-                projectId: 'projectId',
-                ttl: 1695725457,
-                type: 'USER_ATTRIBUTE#sprojectId#appId#userattributename',
-                updateAt: 1695120657125,
-                valueEnum: ['value1', 'value2', 'value3'],
-                valueType: 'String',
+                ...userAttributeDDBCommonData,
+                day7: {
+                  ...userAttributeDayCommonData,
+                  valueEnum: [
+                    {
+                      count: 10,
+                      value: 'value1',
+                    },
+                  ],
+                },
+                day9: {
+                  ...userAttributeDayCommonData,
+                  valueEnum: [
+                    {
+                      count: 20,
+                      value: 'value1',
+                    },
+                  ],
+                },
+                day20: {
+                  ...userAttributeDayCommonData,
+                  valueEnum: [
+                    {
+                      count: 10,
+                      value: 'value1',
+                    },
+                    {
+                      count: 15,
+                      value: 'value2',
+                    },
+                    {
+                      count: 25,
+                      value: 'value3',
+                    },
+                  ],
+                },
+                summary: {
+                  name: 'userattributename',
+                  valueType: 'String',
+                  category: 'category',
+                  valueEnum: [
+                    {
+                      count: 10,
```

Comment:
It is not expected value.

Reply:
make sure it tests the expected function.

Reply:
For user attribute, the summary data is from the latest dayN data, not a aggregation result.